custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -125,7 +125,7 @@
         "type": "expression",
         "transform": {},
         "is_nullable": true,
```

Comment:
yeah, it should. `person_cases.csv` has the same issue. based on the discussion on the delivery forms data source, it sounds like this will be difficult to change?